src/mixxx.cpp

```diff
@@ -1160,7 +1160,7 @@ void MixxxMainWindow::initActions()
 
     for (int i = 0; i < kMaximumVinylControlInputs; ++i) {
         vinylControlTitle.push_back(
-                tr("Enable Vinyl Control &%1").arg(i + 1));
+                tr("Enable Vinyl Control %1").arg(i + 1));
```

Comment:
&%1 works for me on Ubuntu Trusty. Where is it not working? 
It is OK for me to remove it since cursor key navigation works perfectly. 
But you should drop a comment about the issue to net re-iterate the cycle later.